cogs/utils.py

```diff
@@ -0,0 +1,46 @@
+import discord
+from discord.ext import commands
+
+
+class helpCog(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        bot.help_command = sendHelpCommand()
+
+
+class sendHelpCommand(commands.HelpCommand):
```

Comment:
Change this to a slash command (appears in the menu when typing '/') for easier access and make the response ephemeral.

Reply:
Refactor the send_command_help functionality to work with a slash command parameter.